awx_collection/plugins/modules/tower_inventory_source_update.py

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/python
+# coding: utf-8 -*-
+
+# (c) 2020, Bianca Henderson <bianca@redhat.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: tower_inventory_source_update
+author: "Bianca Henderson (@beeankha)"
+short_description: Update inventory source(s).
+description:
+    - Update Ansible Tower inventory source(s). See
+      U(https://www.ansible.com/tower) for an overview.
+options:
+    inventory:
+      description:
+        - Name of the inventory that contains the inventory source(s) to update.
+      required: True
+      type: str
+    inventory_source:
+      description:
+        - The name of the inventory source to update.
+      required: True
+      type: str
+extends_documentation_fragment: awx.awx.auth
+'''
+
+EXAMPLES = '''
+- name: Update a single inventory source
+  tower_inventory_source_update:
+    inventory: "My Inventory"
+    inventory_source: "Example Inventory Source"
```

Comment:
Will this wait until the update has finished? Either way, could we get an example of showing how to do that?

Reply:
I believe she is waiting for the wait from tower_project_update to be merged to add that code.

Reply:
Yep, @sean-m-sullivan is correct!